db/migrate/20220410100413_create_active_storage_tables.active_storage.rb

```diff
@@ -0,0 +1,36 @@
+# This migration comes from active_storage (originally 20170806125915)
```

Comment:
I assume this changes how student-submitted file are stored. Is this backwards-compatible? that is, can we still access the files already in database?

Reply:
I didn't notice any problems in rubyric-test, but it did not have much existing submissions, just few earlier ones by developers.